themes/crisp/translations/pt_BR.json

```
@@ -64,21 +64,21 @@
+    "views.about.jumbotron.text.2": "<strong>Contexto.</strong> A ideia por trás ToS;DR emergiu durante o <a href=\"http://events.ccc.de/camp/2011/wiki/ToS;DR\">Chaos Communication Camp 2011</a> próximo a Berlim, com pessoas do <a href=\"http://unhosted.org\">Unhosted</a> um movimento voltado para a criação de aplicativos web que deem controle aos usuários sobre os seus dados valiosos e sua privacidade. No verão de 2012, <a href=\"http://hugoroy.eu/index.en.html\">Hugo Roy</a> (<a href=\"http://twitter.com/hugoroyd\">@hugoroyd</a>) iniciou a análise legal e trouxe o projeto à vida. Desde então, mais pessoas se juntaram à equipe e contribuíram pela comunidade de revisores. Por fim, todo o trabalho é transparente e as discussões ocorrem em público. O nosso trabalho é <a href=\"thanks\">financiado</a> por organizações sem fins lucrativos e doações individuais e é disponibilizado como <a href=\"http://fsfe.org/about/basics/freesoftware.html\">software livre</a> e dados abertos.",
     "views.about.jumbotron.text.3": "Por favor tenha em mente que o projeto ainda está em fase inicial e a maior parte dos dados está sujeita a alterações importantes. Mais informações sobre a <a href=\"classification\">nossa classificação</a>.",
     "views.about.jumbotron.text.4": "<strong>O que significa “ToS;DR”?</strong> O nome é inspirado pelo acrônimo, em inglês, TL;DR, que significa \"Too Long; Didn't Read\" (em português: “longo demais; não li”) e é frequentemente usado em blogs e e-mails quando um bloco de texto é simplesmente longo demais e as pessoas ficam com preguiça de ler a coisa toda. A intenção era para ser mais um codinome que um nome de verdade. Mas parece que as pessoas gostam dele (e você?)",
     "views.about.title": "Sobre nós",
     "views.api.examples.header": "Exemplos",
     "views.api.examples.text": "O verificador de ToS Checker é <a href=\"https://github.com/shybyte/tos-checker\">uma extensão para Chrome/Chromium</a>, feita por shybyte.",
-    "views.api.feedback.text": "Seria ótimo se você puder nos dar feedback sobre esta página simples da API no Github ou por e-mail. Obrigado!",
+    "views.api.feedback.text": "Seria ótimo se você puder nos dar um feedback através desta simples página da API no Github ou via e-mail. Obrigado!",
```

```suggestion
    "views.api.feedback.text": "Seria ótimo se você puder nos dar um feedback sobre esta simples página da API pelo Github ou via e-mail. Obrigado!",
```

* O feedback é sobre o quê? Sobre a simples página da API.
* Por onde fazê-lo? Pelo Githu ou por e-mail.

O feedback não é através da API.

---

Concordo, eu entendi agora!